c/anagrams_kata/anagrams_finder.h

```diff
@@ -0,0 +1,14 @@
+#ifndef ANAGRAMS_FINDER_H
+#define ANAGRAMS_FINDER_H
+
+typedef struct anagrams_finder_s
+{
+    char *wordlist;
+    void (*find)(struct anagrams_finder_s *_this, char *word);
```

Comment:
You are correct with the two points, which is also why I had started #5 . In my view, a kata should present itself (the starting point!) equally as much as possible in all languages. I'm also with you regarding the reasoning, and would like to (be able to) get there in all variants - after all, the main readme also refers to performance as a point for consideration.


Ok, since we're not meeting tonight, my question about the data structure here.
I see the data structure "public" (meaning: in the header), with the find method as a member. I am curious about this: Why was it made a member (pointing to the static implementation), and not the static function exported?
So far I've seen C-APIs exporting functions via structures only when an extensible interface was anticipated, which furthermore requires versioning/tagging in the structure and so forth.